openstack/orchestration/v1/stackevents/doc.go

```diff
@@ -1,4 +1,19 @@
 // Package stackevents provides operations for finding, listing, and retrieving
 // stack events. Stack events are events that take place on stacks such as
 // updating and abandoning.
+/*
```

Comment:
doc nit: instead of using `/*` here, let's just remove all of the above `//` comments and put `/*` at the very top so we're not using two different styles of comments.

Reply:
Done